rhea/build/boards/xilinx/_ppro.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2015 Christopher Felton, Felix Vietmeyer
+#
+
+from myhdl import TristateSignal, intbv
+
+from rhea.build import FPGA
+from rhea.build.extintf import Port
+from rhea.build.toolflow import ISE
+
+VCCO = 'LVCMOS33'
+
+class PapilioPro(FPGA):
+    vendor = 'xilinx'
+    family = 'spartan6'
+    device = 'XC6SLX9'
+    package = 'TQG144'
+    speed = '-2'
+    version = 1
+    _name = 'pprov'
+    no_startup_jtag_clock = True
+    
+    default_clocks = {
+        'clock': dict(frequency=32e6, pins=(94,),
+                    iostandard='LVTTL')        
+    }
+    
+    default_ports = {
+        'wingA': dict(pins=(48, 51, 56, 58, 61, 66, 67, 75, 79, 81, 83,
```

Comment:
Never mind, I forgot the other papilio board definition uses `wingB` ...  Feel free to leave it (it might be changed in the future but they will all be changed together).